draft-xyz-rats-psa-endorsements.md

```diff
@@ -104,24 +111,44 @@ There are three basic types of PSA endorsements:
 * Certification Claims ({{sec-certificates}}), i.e., metadata that describe
   the certification status associated with a PSA device.
 
+Each type is encoded using a separate CoMID.
+
+## PSA RoT Identification
+{: #sec-impl-id}
+
+Each PSA Endorsement is associated with a well defined immutable PSA RoT.  The
```

Comment:
Perhaps this can be simplified to state that 
PSA Immutable RoT is identified by an unique identifier, known as Implementation ID.

Any PSA Endorsement CoMID associated to this Immutable PSA RoT need to have this identifier MUST be encoded in the top-level `element-name-map`.

This way the text become more logical and straight forward!

Reply:
I agree this is logical and straightforward, but isn't that exactly what it says now?

Reply:
This is just a suggestion!

Reply:
Hmmm, the explanation sounded a bit rounded and complicated for a simple thing! I just attempted to simplify it :)

Reply:
I am completely open to each and every possible simplification.  The problem I'm having is that your suggestion looks exactly the same as the current state.  I am probably wrong but I need you to explain exactly how you want to reflow the current text.